src/main/java/io/temporal/samples/hello/HelloPeriodic.java

```diff
@@ -88,14 +94,38 @@
   // Define the workflow implementation which implements the greetPeriodically workflow method.
   public static class GreetingWorkflowImpl implements GreetingWorkflow {
 
-    // Here we introduce a random delay between periodic executions
+    // In this example, we use an internal of 5 seconds with an intended variation of +/- 1 second
+    // between executions of some useful work. In real applications a higher value may be more
+    // appropriate, for example one that matches a business cycle of several hours or even days.
+    private static final int SCHEDULE_PERIOD_TARGET_SECS = 5;
+    private static final int SCHEDULE_PERIOD_VARIATION_SECS = 2;
+
+    // The max history length of a single Temporal workflow is 50,000 commands.
+    // Therefore, a workflow cannot we cannot run indefinitely. Instead, we use
```

Comment:
"Therefore, a workflow cannot we cannot run indefinitely"

please fix this sentence.

Reply:
Sorry for the typo. 

✔️